cluster/manifests/skipper/routegroup-probe.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: zalando.org/v1
+kind: RouteGroup
+metadata:
+  name: synthetic-probe
+  namespace: default
```

Comment:
maybe use a different namespace?

Reply:
Not that users remove this RouteGroup because it's in `default`.

Reply:
Moved to kube-system